deployment/network-operator/templates/operator.yaml

```diff
@@ -74,6 +74,14 @@ spec:
             - name: HTTPS_PROXY
               value: {{ .Values.proxy.httpsProxy }}
             {{- end }}
+            {{- if .Values.proxy.httpProxy | empty | not }}
```

Comment:
Q: why we use `{{- if .Values.proxy.httpProxy | empty | not }}` instead of just `{{- if .Values.proxy.httpProxy }}`?